container-compose.yml

```diff
@@ -83,16 +92,18 @@ services:
       DATABASE_URL: postgresql://forem:forem@db:5432/PracticalDeveloper_development
     volumes:
       - .:/opt/apps/forem:z
-    entrypoint: ["dockerize", "-wait", "tcp://db:5432", "-wait", "tcp://redis:6379", "-wait", "http://rails:3000", "-timeout", "300s"]
     command: ["bundle", "exec", "rake","db:seed"]
 
   sidekiq:
     image: quay.io/forem/forem:development
     container_name: forem_sidekiq
     depends_on:
-      - rails
-      - redis
-      - db
+      rails:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
+      db:
+        condition: service_healthy
```

Comment:
Due to the use of this `depends_on.[*].condition`, which exists only in v2.4+ and in Compose Spec (but *not* any of the 3.x formats), and due to an above use of `start_period` (which exists in 2.1+, 3.4+, and Compose Spec), let's go ahead and remove the `version: "3"` from the top of this file: there is no v3 format that can make this file valid, and [`version` is deprecated anyway](https://docs.docker.com/compose/compose-file/04-version-and-name/) so there's no point in fixing this anomaly.

Reply:
Hi @klardotsh you absolutely right, I'll remove the version attribute